packages/langium/src/languages/generated/ast.ts

```diff
@@ -25,7 +25,7 @@ export function isAbstractRule(item: unknown): item is AbstractRule {
     return reflection.isInstance(item, AbstractRule);
 }
 
-export type AbstractType = Action | Interface | ParserRule | Type;
```

Comment:
Why was Action removed here?

Reply:
In my eyes, an action is no type. An action changes the type of a node, but an action is no type itself.

Looking at actions and types, there are two different cases:
- The action uses an already declared type (`{WantedType}`). In this case, the action don't need to be covered by AbstractType, since `Interface | Type | ParserRule` are already covered, who declared the type used in the action.
- The action explicitly infers the wanted type now, since the wanted type is not yet declared (`{infer WantedType}`). In this case, the `InferredType` of the action is used as type and submitted to the global index, not the action itself. (Note, that this PR proposes `InferredType`s to be `AbstractType`s. Previously, the action was submitted to the global scope as type.)

Therefore, Actions don't need to be AbstractTypes anymore.

With other words, there are no changes in the behavior of abstract types and actions, only the wording changed.